devops/scripts/install_build_tools.sh

```diff
@@ -16,6 +16,7 @@ apt update && apt install -yqq \
       libffi-dev \
       libva-dev \
       libtool \
+      libdw1 \
```

Comment:
Are all these libraries added for `cm-cpu-emulation`? I think it's worth adding a comment to this script.

Reply:
I do not know that. I can say that `libffi-dev` is used by `cm-cpu-emulation`, but that does not mean it is not used by something else.